### PR TITLE
feat(quickstart): Halyard + GKE on public endpoint

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: setup/quickstart/halyard-gce.md [hunk 1]
```diff
@@ -137,8 +137,8 @@ SSH into the VM with the following ports forwarded as shown to allow access to
 the Spinnaker UI & API servers.
 
 > :warning: You need to SSH into the Halyard host VM from your local
-> workstation; SSHing from [Cloud Shell](https://cloud.google.com/shell/) or
-> another VM won't open the necessary SSH tunnels that will allow your local
+> workstation; SSHing from [Cloud Shell](https://cloud.google.com/shell/), a
+> Chrombook, or another VM won't open the necessary SSH tunnels that will allow your local
```
Comment (duftler) [inline setup/quickstart/halyard-gce.md:141]: s/Chrombook/Chromebook

File: setup/quickstart/halyard-gke-public/index.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+* Spinnaker's API server protected by OAuth2.0
+* Spinnaker's API & UI servers fronted by public IP addresses
+* Subdomains for Spinnaker's API & UI servers
+
+## Part 0: Prerequisites
+
+You need a [GAFW domain](https://admin.google.com) (referred to as `$DOMAIN`
+for the remainder of this tutorial).
+
+You need a running Spinnaker cluster in GKE. If you don't have you, please go
```
Comment (duftler) [inline setup/quickstart/halyard-gke-public/index.md:33]: s/have you/have one

File: setup/quickstart/halyard-gke-public/index.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+   image_path="./edit-id.png"
+   alt="image showing client id edit link"
+%}
+
+{% include figure
+   image_path="./new-redirect.png"
+   alt="image showing updated redirect url"
+%}
+
+Now authorize the UI and API servers to recieve requests at these urls using
```
Comment (duftler) [inline setup/quickstart/halyard-gke-public/index.md:158]: s/recieve/receive

File: setup/quickstart/halyard-gke-public/index.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+```bash
+hal config security ui edit \
+    --override-base-url http://spinnaker.$DOMAIN
+
+hal config security api edit \
+    --override-base-url http://spinnaker-api.$DOMAIN
+```
+
+Now before we finalize these changes by deploying Spinnaker, we need to edit
```
Comment (duftler) [inline setup/quickstart/halyard-gke-public/index.md:169]: s/Now/Now,

**Review by duftler — COMMENTED** [review]
Awesome.
Just a couple of minor typos.

**Comment by lwander** [comment]
Thanks, fixed directly via github web editor.

**Pull request merged**